pkgs/development/python-modules/pydy/default.nix

```diff
@@ -6,47 +6,53 @@
   scipy,
   sympy,
   setuptools,
-  nose,
+  pytestCheckHook,
   cython,
+  fetchpatch2,
 }:
 
 buildPythonPackage rec {
   pname = "pydy";
   version = "0.7.1";
-
   pyproject = true;
-  build-system = [ setuptools ];
 
   src = fetchPypi {
     inherit pname version;
     hash = "sha256-aaRinJMGR8v/OVkeSp1hA4+QLOrmDWq50wvA6b/suvk=";
   };
 
+  build-system = [ setuptools ];
+
+  patches = [
+    # Migrate tests to pytest
```

Comment:
Do we need to remove it at the next bump ? Maybe it would be nice to mention it.

Reply:
@drupol, do you mind if I undraft this and merge? I don’t think we need to block on this as the patch will visibly fail on the next version bump, and it’s one of the few remaining uses of nose.

Reply:
Totally fine!